runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -552,6 +552,12 @@ public void show()
 
 					if (configManager.getConfiguration(CONFIG_GROUP, CONFIG_CLIENT_MAXIMIZED) != null)
 					{
+						// Make sure that the bounds are correct for the current display.
+						// The method only uses the parameter on OSX so passing null is ok.
+						if (OSType.getOSType() == OSType.MacOS)
```

Comment:
do you mean `!=` here...? Your issue/comment/code all disagree with each other

Reply:
Yes actually. Let me fix that real quick.